### PR TITLE
docs(README): README.md cleanup

### PR DESCRIPTION
**Pull request by pmalek**
This PR removes dead links from README, reorganizes sections and links definitions and it also adds some information about our container images and where can they be found.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,30 +20,23 @@ Custom Resource Definitions (CRDs) and Kubernetes-native tooling.
 - **Ingress routing**
   Use [Ingress][ingress] resources to configure Kong.
 - **Enhanced API management using plugins**
-  Use a wide-array of [plugins][kong-hub]
+  Use a wide-array of [plugins][docs-konghq-hub]
```
Comment (rainest) [inline README.md:23]:
```suggestion
  Use a wide array of [plugins][docs-konghq-hub]
```

File: README.md [hunk 2]
```diff
@@ -20,30 +20,23 @@ Custom Resource Definitions (CRDs) and Kubernetes-native tooling.
...
   to monitor, transform, and protect your traffic.
 - **Native gRPC support**
-  Proxy gRPC traffic and gain visibility into it using
-  Kong's plugin.
+  Proxy gRPC traffic and gain visibility into it using Kong's plugin.
```
Comment (rainest) [inline README.md:26]:
```suggestion
  Proxy gRPC traffic and gain visibility into it using Kong's plugins.
```

File: README.md [hunk 3]
```diff
@@ -20,30 +20,23 @@ Custom Resource Definitions (CRDs) and Kubernetes-native tooling.
...
 
 ## Get started
 
-You can use
-[Minikube, Kind](https://kubernetes.io/docs/tasks/tools/)
-on your local machine or use
-a hosted k8s service like
-[GKE](https://cloud.google.com/kubernetes-engine/).
+You can use [Minikube, Kind][k8s-io-tools] on your local machine or use
+a hosted k8s service like [GKE](https://cloud.google.com/kubernetes-engine/).
```
Comment (rainest) [inline README.md:39]:
```suggestion
You can use [Minikube or Kind][k8s-io-tools] on your local machine or use
a hosted Kubernetes service like [GKE](https://cloud.google.com/kubernetes-engine/).
```

**Review by rainest — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -59,84 +52,99 @@ $ helm repo update
...
-All documentation for the Kong Ingress Controller is present in the [kong/docs.konghq.com](https://github.com/kong/docs.konghq.com) repository. Pull Requests are welcome for additions and corrections.
+Release builds of Kong Ingress Controller can be found on Docker Hub in
+[kong/kubernetes-ingress-controller repository][dockerhub-kic]:
 
-## Guides and Tutorials
-
-Please browse through the [guides][guides] to get started and to learn specific ingress controller operations.
+```
+docker pull kong/kubernetes-ingress-controller:latest
+```
```
[the remark this answers is not in the document]
Comment (pmalek) [reply README.md]: Good point! Removed

File: README.md [hunk 5]
```diff
@@ -7,8 +7,8 @@
 
 # Kong Ingress Controller for Kubernetes (KIC)
 
-Use [Kong][kong] for Kubernetes [Ingress][ingress].
-Configure [plugins][kong-hub], health checking,
+Use [Kong][kong-ce] for Kubernetes [Ingress][ingress].
```
[the remark this answers is not in the document]
Comment (pmalek) [reply README.md]: Reverted PTAL

**Review by rainest — APPROVED** [review]
Added a small additional commit to change `[kong]: https://konghq.com/kong-community-edition/` to `[kong]: https://konghq.com/kong` (where the former now redirects to).

**Pull request merged**